packages/comms/tests/wsDFU.spec.ts

```diff
@@ -20,4 +20,14 @@ describe("WsDFU", function () {
             return wu.delete();
         });
     });
+
+    it.skip("recursiveFetch", async function () {
```

Comment:
wanted to double-check, this test will be skipped. also, there's a console.log at the end of it

Reply:
Skip is deliberate, the console is not!